cockroachdb_dedicated/manifest.json

```diff
@@ -50,4 +50,4 @@
       "source": "cockroach-cloud"
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
Could you please undo this change in a different directory? Thanks :)

Reply:
Oops, done. It was my previous employer haha, so that's where I was cargo-culting from.